src/components/Amenities/Amenities.jsx

```diff
@@ -11,50 +11,73 @@ import {
   PetsIcon,
 } from "../../icons";
 import styles from "./Amenities.module.css";
+import { useState } from "react";
+
+const ICONS_MAP = {
+  kitchen: <KitchenIcon />,
+  workspace: <WorkspaceIcon />,
+  sauna: <SaunaIcon />,
+  balcony: <BalconyIcon />,
+  fireplace: <FireplaceIcon />,
+  wifi: <WifiIcon />,
+  parking: <ParkingIcon />,
+  pets: <PetsIcon />,
+  backyard: <BackyardIcon />,
+  firepit: <FirepitIcon />,
+};
 
 function Amenities({ title, amenities, onClick }) {
```

Comment:
do we need the  onclick prop here since we have it implented in amenities..

Reply:
Removed the onClick prop.